Library/Homebrew/cmd/update-report.rb

```diff
@@ -250,10 +250,24 @@ def migrate_tap_migration
       tabs = dir.subdirs.map { |d| Tab.for_keg(Keg.new(d)) }
       next unless tabs.first.tap == tap # skip if installed formula is not from this tap.
       new_tap = Tap.fetch(new_tap_name)
-      new_tap.install unless new_tap.installed?
-      # update tap for each Tab
-      tabs.each { |tab| tab.tap = new_tap }
-      tabs.each(&:write)
+      # For formulae migrated to cask: Auto-install cask or provide install instructions.
+      if new_tap_name == "caskroom/cask"
+        system "brew", "uninstall", name
```

Comment:
[Like in `Library/brew.rb`](https://github.com/Homebrew/brew/blob/b05a596d570c8e5686fbb45331da8f101179f49a/Library/brew.rb#L114) and other places, `"brew"` needs to be replaced with `HOMEBREW_BREW_FILE`.